pkg/bytecode/vm_test.go

```diff
@@ -43,19 +43,59 @@ func TestVMGlobals(t *testing.T) {
 			wantBytecode: &Bytecode{
 				Constants: makeValues(t, 1, 1),
 				Instructions: makeInstructions(
+					// x := 1
 					mustMake(t, OpConstant, 0),
 					mustMake(t, OpSetGlobal, 0),
+					// y := x
 					mustMake(t, OpGetGlobal, 0),
 					mustMake(t, OpSetGlobal, 1),
+					// y = x + 1
 					mustMake(t, OpGetGlobal, 0),
 					mustMake(t, OpConstant, 1),
 					mustMake(t, OpAdd),
 					mustMake(t, OpSetGlobal, 1),
+					// y = y
 					mustMake(t, OpGetGlobal, 1),
 					mustMake(t, OpSetGlobal, 1),
 				),
 			},
 		},
+		{
+			name: "index assignment",
```

Comment:
Could we please add a double nested array test for left and right hand sight or is this not yet supported?

Reply:
Added a ["nested index assignment"](https://github.com/evylang/evy/pull/318/files#diff-1388e4bf492d933fa753d90c0d447ca9934dabcf4dde2fd61acbda470f35292bR100) test that covers nested assignment and access